homeworks/05-css-advanced/styles/style.css

```diff
@@ -0,0 +1,470 @@
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @media(min-width: 601px) {
+        height: 5rem;
+        width: 5rem;
+        margin-top: 2.0625rem;
+    }
+}
+
+.content__btn-toggle img {
+    @media(min-width: 601px) {
+        height: 2.5rem;
+        width: 2.5rem;
+    }
+}
+
+header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @media(min-width: 601px) {
+        flex-direction: row;
+        gap: 2.5rem;
+        justify-content: center;
+        margin-top: 2.1875rem;
+    }
+}
+
+figure {
+    margin: 0;
+}
+
+.header__picture-avatar {
+    width: 6.6875rem;
+    height: 6.6875rem;
+    padding: 0.25rem;
+    border: 0.25rem solid #FFE071;
+    border-radius: 50%;
+
+    @media(min-width: 601px) {
+        width: 12rem;
+        height: 12rem;
+        padding: 0.5rem;
+        background-color: #fff;
+    }
+}
+
+.header__intro {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 2.125rem;
+
+    @media(min-width: 601px) {
+        align-items: flex-start;
+        gap: 0.75rem;
+        margin-top: 0;
+    }
+}
+
+.intro__title {
+    margin: 0;
+    font-size: 2rem;
+    line-height: 3rem;
+    color: #F1F2F4;
+
+    @media(min-width: 601px) {
+        font-size: 2.875rem;
+    }
+}
+
+.header__intro p {
+    margin: 0;
+    font-size: 0.875rem;
+    line-height: 1.3125rem;
+    color: #A3ABB2;
+
+    @media(min-width: 601px) {
+        font-size: 1.375rem;
+    }
+}
+
+.intro__socialmedia {
+    display: flex;
+    align-items: center;
+    gap: 1.625rem;
+    height: 1.25rem;
+    margin-top: 1.5rem;
+
+    @media(min-width: 601px) {
+        height: 2.25rem;
+        margin-top: 0.25rem;
+        gap: 1.875rem;
+    }
+}
+
+.intro__icon {
+    height: 1.5rem;
+    width: 1.5rem;
+
+    @media(min-width: 601px) {
+        width: 2rem;
+        height: 2rem;
+    }
+}
+
+.list {
+    display: flex;
+    align-items: flex-start;
+    padding: 1.25rem;
+    justify-content: space-between;
+    margin: 0 auto;
+
+    @media(min-width: 601px) {
+        width: 100%;
+    }
+}
+
+.list__item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 6.375rem;
+
+    @media(min-width: 601px) {
+        width: 12.125rem;
+    }
+}
+
+.list__item-title {
+    margin: 0;
+    color: #C3C7CB;
+    font-size: 1rem;
+    line-height: 1.5rem;
+
+    @media(min-width: 601px) {
+        font-size: 1.5rem;
+        line-height: 2.25rem;
+        color: #A3ABB2;
+    }
+}
+
+.list__item p {
+    margin: 0;
+    color: #A3ABB2;
+    font-size: 0.875rem;
+    line-height: 1.3125rem;
+    text-align: center;
+
+    @media(min-width: 601px) {
+        font-size: 1.375rem;
+        line-height: 2.0625rem;
+        content: "Satisfied customers";
+        width: 12.125rem;
+    }
+}
+
+.content__cv-contact {
+    display: flex;
+    justify-content: center;
+    gap: 0.875rem;
+    margin: 1.25rem auto;
```

Comment:
You are adjusting 1 between elements distance with 1el-t bottom-margin, next el top margin and having flex gap for their container for that place.
It is much better to use 1 parameter. Also such distances should be up to design(in pixels) for all screens.